src/updatechecker.cpp

```diff
@@ -41,8 +42,14 @@ void UpdateChecker::autoCheck()
 
 }
 
-void UpdateChecker::check(bool silent)
+void UpdateChecker::check(bool silent, int currentComboBoxUpdateLevel)
 {
+	// catch value if possible, s. comment at start of checkForNewVersion
+	if (currentComboBoxUpdateLevel == -1)
+		comboBoxUpdateLevel = -1;
+	else
+		comboBoxUpdateLevel = currentComboBoxUpdateLevel;
+

```

Comment:
this makes no sense, it is basically `comboBoxUpdateLevel = currentComboBoxUpdateLevel;`